libraries/AP_HAL_Linux/Thread.cpp

```diff
@@ -55,6 +55,12 @@ bool Thread::_run()
     return true;
 }
 
+/* Round up to the specified alignment */
```

Comment:
ugh... if it was a generic function to be added in another place, yes.  Not here.